commit 0b3d4242b31fd3773bf58152efde0d4952436ded
Author: Casey Schaufler <casey@schaufler-ca.com>
Date:   Thu Aug 3 16:51:40 2017 -0700

    UBUNTU: SAUCE: LSM stacking: LSM: Complete task_alloc hook
    
    The Task alloc hook needs to allocate the data.
    
    Signed-off-by: Casey Schaufler <casey@schaufler-ca.com>
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/security/security.c b/security/security.c
index c48d7c46cc0a..0c5c92fefb63 100644
--- a/security/security.c
+++ b/security/security.c
@@ -1365,6 +1365,10 @@ int security_file_open(struct file *file, const struct cred *cred)
 
 int security_task_alloc(struct task_struct *task, unsigned long clone_flags)
 {
+	int rc = lsm_task_alloc(task);
+
+	if (rc)
+		return rc;
 	return call_int_hook(task_alloc, 0, task, clone_flags);
 }
 
